pyproject.toml

```diff
@@ -32,63 +32,63 @@ packages = []
 
 [tool.poetry.dependencies]
 python = "^3.9"
-beautifulsoup4 = "4.9.0"
-boto3 = "1.12.26"
-celery = "5.2.2"
-celery-redbeat = "2.0.0"
-cryptography = ">=39.0,<40.0"
-dj-database-url = "0.3.0"
-dj-static = "0.0.6"
-django = "2.2.28"
-django-compat = "1.0.15"
+beautifulsoup4 = "^4.9.0"
+boto3 = "^1.12.26"
+celery = "^5.2.2"
+celery-redbeat = "^2.0.0"
+cryptography = "^39.0"
+dj-database-url = "^0.3.0"
+dj-static = "^0.0.6"
+django = "^2.2.28"
+django-compat = "^1.0.15"
 django-elastic-transcoder = {git = "https://github.com/mitodl/django-elastic-transcoder", tag = "v0.9.7"}
 django-encrypted-model-fields = "^0.6.5"
-django-hijack = "2.1.10"
-django-hijack-admin = "2.1.10"
-django-redis = "5.2.0"
-django-shibboleth-remoteuser = {git = "https://github.com/Brown-University-Library/django-shibboleth-remoteuser", tag = "v0.9"}
-django-webpack-loader = "0.5.0"
-djangorestframework = "3.11.2"
-google-api-python-client = ">=2.58.0"
-google-auth = ">=2.11.0"
-google-auth-oauthlib = ">=0.5.2"
-html5lib = "0.999999999"
+django-hijack = "^2.1.10"
+django-hijack-admin = "^2.1.10"
+django-redis = "^5.2.0"
+django-shibboleth-remoteuser = {git = "https://github.com/Brown-University-Library/django-shibboleth-remoteuser", tag = "v0.12"}
+django-webpack-loader = "^0.5.0"
+djangorestframework = "^3.11.2"
+google-api-python-client = "^2.58.0"
+google-auth = "^2.11.0"
+google-auth-oauthlib = "^0.5.2"
+html5lib = "^0.999999999"
 httplib2 = "^0.22.0"
-ipython = "7.16.3"
+ipython = "^7.16.3"
 mit-moira = {git = "https://github.com/mitodl/mit-moira", tag = "v0.0.4"}
 newrelic = "^8.8.0"
-psycopg2 = "2.8.6"
+psycopg2 = "^2.8.6"
 pycountry = "^22.3.5"
-python-dateutil = "2.8.1"
+python-dateutil = "^2.8.1"
 pytz = "^2023.3"
-redis = "4.4.4"
-requests = ">=2.20.0"
-sentry-sdk = "0.14.3"
-smart-open = "1.5.7"
-structlog = "20.1.0"
-structlog-sentry = "1.2.2"
-urllib3 = ">=1.24.2"
+redis = "^4.4.4"
+requests = "^2.20.0"
+sentry-sdk = "^0.14.3"
+smart-open = "^1.5.7"
+structlog = "^20.1.0"
+structlog-sentry = "^1.2.2"
+urllib3 = "^1.24.2"
 uwsgi = "2.0.21"
 
 
 [tool.poetry.group.dev.dependencies]
-black = "22.8.0"
```

Comment:
Can we keep this pegged to a particular version, so as to avoid potential failures when the default `black` criteria change?